arch/arm/configs/aspeed_defconfig

```diff
@@ -148,7 +151,7 @@ CONFIG_USB_STORAGE=y
 CONFIG_USB_SERIAL=y
 CONFIG_USB_SERIAL_GENERIC=y
 CONFIG_USB_GADGET=y
-CONFIG_USB_G_SERIAL=y
+CONFIG_USB_G_SERIAL=m
 CONFIG_MMC=y
```

Comment:
On Tue, Oct 13, 2015 at 4:42 PM, apopple <notifications@github.com> wrote:
> Just talked Cyril ... no reason really. Apparently the kernel defconfig generator made the change. Is this something we care about?

Not really, it's just for the cases where we're netbooting kernels we
will be missing any modules, so we want everything to be either =y or
=n. I don't think anyone is doing usb stuff yet.